src/python/model/common/metric.py

```diff
@@ -0,0 +1,21 @@
+import torch
+from sklearn.metrics import multilabel_confusion_matrix
+
+
+class Metric:
```

Comment:
Please, add some descriptions for this class. Maybe is it better to call it Measurer?
Moreover, it looks that you should have several fields in this class:
- threshold
- predictions
- targets
In init method you could get threshold and evaluation_predictions

- `get_f1_score` should be private (marked __)
- for `f1_score` in compute_metric result please create an enum class with keys of the mesuares

Reply:
can I leave `get_f1_score` public as I use the method during [evaluation](https://github.com/hyperskill/hyperstyle/blob/roberta-model/src/python/evaluation/qodana/imitation_model/evaluation.py#L46) (model outputs differ on training and evaluation stage – I thought it would be good to separate computation into 2 parts)

Reply:
and I think I didn't get what are the **keys of the mesuares** :( 
could you explain, please?

firstly I thought it should be something like:
@unique
class MeasurerArgument(Enum):
    FP = 'false_positives'
    FN = 'false_negatives'
    TP = 'true_positives'
    TN = 'true_negatives'
but it looks like we compute values in method and there's no need to store just names in enum class

Reply:
Yes, I meant something like this for the returned dictionary in the compute_metric function:
`return {'f1_score': self.get_f1_score(predictions, torch.tensor(targets))}` -> `return {MeasurerArgument.F1_score.value: self.get_f1_score(predictions, torch.tensor(targets))}`

Then you can get necessary values from this dict by consts, not by strings